docs/cli/getting-started.mdx

```diff
@@ -58,11 +46,19 @@ For Windows, you will add it to the `%PATH%`.
 
 To manually install Botkube CLI:
 
-1. Download the binary for the version and platform of your choice from the [GitHub releases page](https://github.com/kubeshop/botkube/releases).
+1. Download the binary for the version and platform of your choice from the [GitHub releases page](https://github.com/kubeshop/botkube/releases). Alternately, run:
+
+   ```shell
+   export OS="" # allowed values: darwin, linux, windows
+   export ARCH="" # allowed values: amd64, arm64, armv7
+   #export SUFFIX=".exec" # uncomment if OS is `windows`
+   curl -Lo botkube https://github.com/kubeshop/botkube/releases/download/v1.1.1/botkube_${OS}_${ARCH}${SUFFIX}
+   ```
+
 2. Move it to a location in the PATH. For example:
 
    ```shell
-   mv botkube_linux_amd64 /usr/local/bin/botkube
+   mv botkube /usr/local/bin/botkube
```

Comment:
Will the bin already have exec permissin?

Reply:
good point, I restored `chmod +x botkube &&`